constants.js

```diff
@@ -93,66 +93,66 @@ export const tracks: Tracks = {
   "DATA_ENGINEERING": {
     "displayName": "Data engineering",
     "category": "A",
-    "description": "Develops expertise in native mobile platform engineering, such as iOS or Android",
+    "description": "Develops expertise in Data Engineering technologies and procedures such as Big Data, Spark, ETL, Python and AWS",
     "milestones": [{
-      "summary": "Works effectively within established iOS or Android architectures, following current best practices",
+      "summary": "Works effectively within established Big Data and ETL architectures, following current best practices",
       "signals": [
         "Delivers features requiring simple local modifications",
-        "Adds simple actions that call server endpoints",
+        "Adds simple actions that uses the data lake, calls APIs or uses databases",
         "Reuses existing components appropriately",
       ],
       "examples": [
-        "Added existing button to a different iOS surface",
-        "Add follow button for publications on Android",
-        "Fetched and displayed a new stream, using existing stream item styles",
+        "Added a new enrichment to the ETL",
```

Comment:
Maybe adding a new enrichment from scratch is an example of a slightly higher milestone?

```suggestion
        "Updated an existing enrichment ETL",
```